cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js

```diff
@@ -1176,4 +1171,27 @@ export default {
     );
     cy.expect(Popover({ content: including(message) }).exists());
   },
+
+  verifyPaymentStatusDropdownOptions: (...names) => {
+    cy.get('#po-line-details')
+      .find('label:contains("Payment status") + div button[aria-haspopup]')
```

Comment:
can't we get this element with interactors?
if not, please make this as a variable

Reply:
Hi, unfortunately this can not be done with interactors, I've tried. Element is very generic and there are couple more similar dropdowns with 'Accepted Value' label in the same section.
Speaking about variable, 'Cypress' not allows to define it one time and use.